dev-resources/template/1_db.yml

```diff
@@ -28,10 +27,11 @@ Parameters:
     Default: "12.9"
     AllowedValues:
       - "12.9"
+      - "14.3"
   DBInstanceClass:
     Description: For provisioned aurora, the instance class to use
     Type: String
-    Default: "db.r4.large"
+    Default: "db.t4g.medium"
```

Comment:
What's the rationale behind the change in instance size?

Reply:
addressed via https://github.com/yetanalytics/lrsql/pull/276/commits/c2acf07e55fd5860527b8919683e4c5fa1af35a6